examples/gstreamer-player/egl_integration.rs

```diff
@@ -3,13 +3,15 @@
 
 use std::sync::{Arc, Mutex};
 
+use futures::channel::mpsc::UnboundedSender;
 use gst::prelude::*;
 use gst_gl::prelude::*;
 
 pub fn init<App: slint::ComponentHandle + 'static>(
     app: &App,
     pipeline: &gst::Pipeline,
     new_frame_callback: fn(App, slint::Image),
+    bus_sender: &UnboundedSender<gst::Message>,
```

Comment:
I suggest to pass ownership here. In practice there's only one bus, isn't there?